File: easy/math/Palindrome Number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        string s = to_string(x);
        string r ="";
        for(int i=s.length()-1;i>=0;i--){
            r+=s[i];
        }
        if(r==s){return true;}
        return false;
    }
};

File: easy/math/Palindrome Number.java
class Solution {
    public boolean isPalindrome(int x) {
        String s = String.valueOf(x);
        String r ="";
        for(int i=s.length()-1;i>=0;i--){
            r+=s.charAt(i);
        }
        if(r.equals(s)){return true;}
        return false;
    }
}
